### PR TITLE
add liquidity swap price impact

### PR DESCRIPTION
**Pull request by vmaark**
Add price impact to add/remove liquidity, fix error on swap tab (related to price impact calc)

### CODE REVIEW

**Comment by kyriediculous** [comment]
Looks like there's some build errors

File: packages/app/src/utils/tenderSwapHooks.ts [hunk 1]
```diff
@@ -61,18 +60,59 @@ export const useCalculateRemoveLiquidityOneToken = (
...
+) => {
+  if (tokenInputAmount.lte(0)) {
+    return constants.Zero;
+  }
+
+  return isWithdraw
+    ? (weiToEthInFloat(tokenOutputAmount) / (weiToEthInFloat(tokenInputAmount) * weiToEthInFloat(virtualPrice)) - 1) *
+        100
+    : ((weiToEthInFloat(virtualPrice) * weiToEthInFloat(tokenOutputAmount)) / weiToEthInFloat(tokenInputAmount) - 1) *
+        100;
```
Comment (kyriediculous) [inline packages/app/src/utils/tenderSwapHooks.ts:105]: I think percentages should be returned as a decimal representation (i.e. 5% is 0.05) and only be converted to % for display on the front-end in the component itself.

File: packages/app/src/utils/tenderSwapHooks.ts [hunk 2]
```diff
@@ -88,21 +88,19 @@ export const useSwapPriceImpact = (
...
   return isWithdraw
-    ? (weiToEthInFloat(tokenOutputAmount) / (weiToEthInFloat(tokenInputAmount) * weiToEthInFloat(virtualPrice)) - 1) *
-        100
-    : ((weiToEthInFloat(virtualPrice) * weiToEthInFloat(tokenOutputAmount)) / weiToEthInFloat(tokenInputAmount) - 1) *
-        100;
+    ? weiToEthInFloat(tokenOutputAmount) / (weiToEthInFloat(tokenInputAmount) * weiToEthInFloat(virtualPrice)) - 1
+    : (weiToEthInFloat(virtualPrice) * weiToEthInFloat(tokenOutputAmount)) / weiToEthInFloat(tokenInputAmount) - 1;
 };
 
 export const useLiquidityPriceImpact = (
```
Comment (kyriediculous) [inline packages/app/src/utils/tenderSwapHooks.ts:106]: this can be removed, no ?

**Review by kyriediculous — CHANGES_REQUESTED** [review]

File: packages/app/src/utils/tenderSwapHooks.ts (hunk 2 above)
Comment (vmaark) [reply packages/app/src/utils/tenderSwapHooks.ts:106]: that's what we use on add/remove

**Review by kyriediculous — APPROVED** [review]

**Pull request merged**